packages/charts/src/components/common/components/Tooltip/helpers/getTooltipEntries.ts

```diff
@@ -35,8 +34,7 @@ export const getTooltipEntries = (
                 title: key,
                 value: getDataPointValue(missingValueLabel, datumByKey[key]?.datum.value, valueFormatter),
                 color: colorAccessor(key),
-
-                ...(valueContextBySeries?.[index] ? { valueContextBySeries: valueContextBySeries?.[index] } : {}),
+                ...(datumByKey[key]?.datum.valueContext ? { valueContext: datumByKey[key]?.datum.valueContext } : {}),
```

Comment:
is this not functionally equivalent to simply doing

```suggestion
                valueContext: datumByKey[key]?.datum.valueContext,
```
since if `datumByKey[key]?.datum.valueContext` is undefined it results in that key of the object being undefined if it is assigned directly, just like it would be if we assign nothing at all by spreading an empty object.

Reply:
No difference, I just find it cleaner to pass the property if it needs the value, because it's a common component shared across charts. But I'll change it, since you guys are now responsible for this code 😄

Reply:
Don't do it because we're responsible, do it because it's cleaner (**if you agree**). 

If you prefer not passing anything at all, which I can understand, you can always move the object to a separate variable excluding the `valueContext` upon its initialization and then conditionally adding a the `valueContext` key before pushing the object into the array. I think we can both agree that combining ternaries with the spread operator and optional chaining results in some pretty esoteric code and dummies like myself will have a harder time reading it than plain KISS-style programming 😅